doc/glossary.md

```diff
@@ -685,7 +685,6 @@ contains two partitions.
   <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/OpenINF/GitHub-Markdown/HEAD/blockquotes/badge/light-theme/info.svg">
   <img alt="Note" src="https://raw.githubusercontent.com/OpenINF/GitHub-Markdown/HEAD/blockquotes/badge/dark-theme/info.svg">
 </picture><br />
```

Comment:
```suggestion
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/OpenINF/GitHub-Markdown/HEAD/blockquotes/badge/light-theme/info.svg">
  <img alt="Note" src="https://raw.githubusercontent.com/OpenINF/GitHub-Markdown/HEAD/blockquotes/badge/dark-theme/info.svg">
</picture><br />
```

Reply:
yeah, it is big;

we have a hacky alternative;

- https://gist.github.com/DerekNonGeneric/94b96be75962fef02fbbddfa57264d0c